REFERENCE.md

```diff
@@ -83,10 +83,13 @@ The `sample.fraction` parameter is a number in the range (0, 1] that controls th
 
 #### `num.trees`
 
-The parameter `num.trees` controls how many trees are grown during training, and defaults to 2000. Generally, obtaining high-quality confidence intervals requires growing more trees than are needed for accurate predictions.
+The parameter `num.trees` controls how many trees are grown during training, and defaults to 2000. Generally, obtaining tighter confidence intervals requires growing more trees than are needed for accurate predictions, and we recommend that users grow trees in proportion to the number of number of observations. If the number of trees in a forest is too small, the confidence intervals can be too wide, and therefore too conservative.
```

Comment:
typo: 'number of number of'.

I think it would be nice to talk about `excess.error` here instead of under 'Merging Forests', so all discussion of how the variance estimates relate to the number of trees can be found in one place. This is just a personal preference though, happy to go with whatever presentation you think is best.

Reply:
I reworded this paragraph substantially and added a little information about `excess.error`.